content/W2015/W15-blog-caleybrock.md

```diff
@@ -14,13 +14,13 @@ Despite her involvement in mathletes and engineering club in high school, Caley
 
 Her passion for design led her to the arts and business program, in which she majored in speech communication with a specialization in digital arts. Dissatisfied with her program, she began taking math courses.
```

Comment:
I think "Arts and Business" is usually capitalized; also, you might want to indicate it's "at the University of Waterloo", since this blog could be read by anyone, anywhere!